src/types/kakao.maps.d.ts

```
@@ -0,0 +1,11 @@
+declare namespace kakao {
+  export namespace maps {
+    class Map {
```

Biome (2.1.2)

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)